src/tree_builder/interface.rs

```diff
@@ -72,6 +72,10 @@ pub trait TreeSink {
     /// Do two handles refer to the same node?
     fn same_node(&self, x: Self::Handle, y: Self::Handle) -> bool;
 
+    /// Are two handles present in the same tree
+    /// https://html.spec.whatwg.org/multipage/infrastructure.html#home-subtree
```

Comment:
This link isn't valid any more. The text in step 12 of https://html.spec.whatwg.org/multipage/syntax.html#create-an-element-for-the-token just says "same tree" with a link to https://dom.spec.whatwg.org/#concept-tree; we can probably just name this `same_tree` in that case.